### PR TITLE
[dev] generate new certificates

### PR DESCRIPTION
**Pull request by OlivierChirouze**
+ nodemon watches whole project

### CODE REVIEW

File: paf-mvp-demo-express/nodemon.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "watch": ["./"],
+  "watch": ["../"],
```
Comment (OlivierChirouze) [inline paf-mvp-demo-express/nodemon.json:2]: This should be a dedicated commit, my bad I was too lazy.
The idea is that since we are using `ts-node` to run the project locally, we need to restart it anytime something changes _in any of the sub-projects_, not just the demo project.

**Review by RomainLofaso — APPROVED** [review]
